src/incubator/TextField/FloatingPlaceholder.tsx

```diff
@@ -67,6 +67,7 @@ const FloatingPlaceholder = ({
 
   return (
     <View absF style={hidePlaceholder && styles.hidden}>
+      {/* @ts-expect-error TS error is caused due to animatedStyle passed to style prop */}
```

Comment:
Not sure it will help but did you try changing the Text's style type to `Animated.AnimatedProps<**StyleProp**<TextStyle>**>**`?

Reply:
I think the way to go is (in `Text`):
```
export type TextProps = Animated.AnimatedProps<RNTextProps>...
```
and remove:
```
style?: StyleProp<TextStyle | Animated.AnimatedProps<TextStyle>>;
```
from there

Reply:
It didn't work :(

Reply:
try: 
`        style={[styles.placeholder, floatingPlaceholderStyle, animatedStyle] as TextStyle}
`

Reply:
try:
`style={[styles.placeholder, floatingPlaceholderStyle, animatedStyle] as TextStyle}
`

Reply:
> It didn't work :(

Very weird, it worked for me locally...

Reply:
@M-i-k-e-l Are you sure? Try doing it and run `npm run build:dev` don't depend on the vscode editor

@lidord-wix 
This is basically the same as ignoring, I prefer to add `ts-expect-error` with a commet, maybe the issue will be resolved in the future.

Reply:
Retested now, pretty sure I clean-installed before testing this branch; I hope it's not my env, can you double check please?